external/webserver/http_client.c

```diff
@@ -215,7 +215,11 @@ int http_parse_message(char *buf, int buf_len, int *method, char *url,
 				buf[sentence_end] = '\0';
 				if (strlen(buf + len->sentence_start) > 0) {
 					/* Read parameters */
-					http_separate_keyvalue(buf + len->sentence_start, key, value);
+					int result = http_separate_keyvalue(buf + len->sentence_start, key, value);
```

Comment:
http_seperate_keyvalue always returns  HTTP_OK.   May i know the reason for checking this error case?   If you foresee a problem situation where keyvalue can't be seperated, then you may have to modify the  http_seperate_keyvalue() API to handle such error case and return HTTP_ERROR

Reply:
http_seperate_keyvalue() returns HTTP_ERROR when the key and value buffer sizes are over  HTTP_CONF_MAX_KEY_LENGTH and HTTP_CONF_MAX_VALUE_LENGTH.